dot/core/messages.go

```diff
@@ -70,32 +74,34 @@ func (s *Service) HandleTransactionMessage(peerID peer.ID, msg *network.Transact
 		return false, err
 	}
 
-	allTxnsAreValid := true
+	allTxsAreValid := true
 	for _, tx := range txs {
-		txnIsValid := true
+		isValidTxn := true
 		validity, err := s.validateTransaction(head, rt, tx)
 		if err != nil {
-			txnIsValid = false
-			allTxnsAreValid = false
-			switch err := err.(type) {
+			switch err.(type) {
```

Comment:
Whenever `err != nil` you should update the `allTxsAreValid` to false
```suggestion
			allTxsAreValid = false
			switch err.(type) {
```

Reply:
This does already happen, but I think this way if cleaner